package.json

```diff
@@ -89,9 +89,10 @@
     "lint:objc": "node ./script/lint.js --objc",
     "lint:py": "node ./script/lint.js --py",
     "lint:gn": "node ./script/lint.js --gn",
-    "lint:docs": "remark docs -qf && npm run lint:js-in-markdown && npm run create-typescript-definitions && npm run lint:ts-check-js-in-markdown && npm run lint:docs-fiddles && npm run lint:docs-relative-links && npm run lint:markdownlint",
+    "lint:docs": "remark docs -qf && npm run lint:js-in-markdown && npm run create-typescript-definitions && npm run lint:ts-check-js-in-markdown && npm run lint:docs-fiddles && npm run lint:docs-relative-links && npm run lint:markdown && npm run lint:markdownlint",
     "lint:docs-fiddles": "standard \"docs/fiddles/**/*.js\"",
     "lint:docs-relative-links": "electron-lint-markdown-links --root docs \"**/*.md\"",
+    "lint:markdown": "node ./script/lint.js --md",
     "lint:markdownlint": "electron-markdownlint \"*.md\" \"docs/**/*.md\"",
```

Comment:
Is there any way we can run `lint:markdownlint` programmatically from within `lint:markdown`? Having two separate scripts is a bit confusing out of context. 😅

Reply:
It's feasible, yea, but more refactoring than I'd like to do in this PR. Follow-up task?

Reply:
Refactor ended up being simpler than I expected, I've included it here.